eq-publisher/src/eq_schema/builders/confirmationPage/ConfirmationPage.js

```diff
@@ -66,6 +66,10 @@ const buildAuthorConfirmationQuestion = (
   const confirmationQuestionObject = {
     id: `confirmation-page-for-${page.id}`,
     title: page.confirmation.title,
+    description:
+      page.answers[0].type === "Checkbox"
```

Comment:
Do we have a constant for this. Or not in `publisher`?

Reply:
Not in Publisher, probably worth adding them though.